compass/ocean/tests/overflow/initial_state.py

```diff
@@ -0,0 +1,84 @@
+import xarray
+import numpy
+import matplotlib.pyplot as plt
+
+from mpas_tools.planar_hex import make_planar_hex_mesh
+from mpas_tools.io import write_netcdf
+from mpas_tools.mesh.conversion import convert, cull
+
+from compass.step import Step
+from compass.model import run_model
+
+
+class InitialState(Step):
+    """
+    A step for creating a mesh and initial condition for overflow test
+    cases
+    """
+    def __init__(self, test_case):
+        """
+        Create the step
+
+        Parameters
+        ----------
+        test_case : compass.ocean.tests.overflow.default.Default
```

Comment:
```suggestion
        test_case : compass.testcase.Testcase
```
You'll need to use the base class since you're passing either `Default` or `RpeTest`.

Reply:
Resolved by 6d716589